src/solutions/ode_solutions.jl

```diff
@@ -129,7 +129,12 @@ function build_solution(prob::Union{AbstractODEProblem, AbstractDDEProblem},
                         interp = LinearInterpolation(t, u),
                         retcode = :Default, destats = nothing, kwargs...)
     T = eltype(eltype(u))
-    N = length((size(prob.u0)..., length(u)))
+
+    if prob.u0 === nothing
+        N = 2
+    else
+        N = length((size(prob.u0)..., length(u)))
```

Comment:
Isn't this the same as `N = ndims(prob.u0) + 1`? Am I missing something?

Reply:
Yes, but this is from yee olde' days of like Julia v0.5, I found that this worked with type inference while `N = ndims(prob.u0) + 1` did not. I'll check it some other time, but it's never come up 😅